roles/influxdb_exporter/meta/argument_specs.yml

```diff
@@ -0,0 +1,67 @@
+---
+# yamllint disable rule:line-length
+argument_specs:
+  main:
+    short_description: "Prometheus Influxdb Exporter"
+    description:
+      - "Deploy prometheus L(influxdb exporter,https://github.com/prometheus/influxdb_exporter) using ansible"
+    author:
+      - "Prometheus Community"
+    options:
+      influxdb_exporter_version:
+        description: "influxdb exporter package version. Also accepts latest as parameter."
+        default: "0.11.5"
+      influxdb_exporter_skip_install:
+        description: "influxdb exporter installation tasks gets skipped when set to true."
+        type: bool
+        default: false
```

Comment:
```suggestion
```

We don't support changing the install behavior with a variable anymore, users can use tags instead, such as `--skip tags install`